### ISSUE TITLE
not showing dropdown

### ISSUE THREAD

**Issue by deblis** [summernote/angular-summernote#143]
Hello

I am doing a proyect with angular and the theme use summernote with joing this js not work  the dropdown when i refresh the page, not display te dropdown. I dont know that happend.
![summernote](https://cloud.githubusercontent.com/assets/7839871/16747506/5fe80848-478d-11e6-853b-47bfcc2cd500.png)
![summernote1](https://cloud.githubusercontent.com/assets/7839871/16747578/a28d104e-478d-11e6-8b0e-20b215970468.jpg)

<script src="views/assets/plugins/jquery/jquery-2.1.4.min.js"></script>
 <script src="views/assets/plugins/jquery-ui/jquery-ui.min.js"></script>
<script src="views/assets/plugins/pace-master/pace.min.js"></script>
<script src="views/assets/plugins/jquery-blockui/jquery.blockui.js"></script>

<script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
<script src="bower_components/summernote/dist/summernote.js" type="text/javascript"></script>

 <script src="views/assets/plugins/jquery-slimscroll/jquery.slimscroll.min.js"></script>
<script src="views/assets/plugins/3d-bold-navigation/js/modernizr.js"></script>
<script src="views/assets/plugins/switchery/switchery.min.js"></script>
<script src="views/assets/plugins/uniform/jquery.uniform.min.js"></script>
<script src="views/assets/plugins/offcanvasmenueffects/js/classie.js"></script>
<script src="views/assets/plugins/offcanvasmenueffects/js/snap.svg-min.js"></script>
<script src="views/assets/plugins/offcanvasmenueffects/js/main.js"></script>
<script src="views/assets/plugins/waves/waves.min.js"></script>
<script src="views/assets/plugins/3d-bold-navigation/js/main.js"></script>

 

<script src="views/assets/plugins/bootstrap-datepicker/js/bootstrap-datepicker.js"></script>
<script src="views/assets/js/modern.min.js"></script>

<script src="node_modules/angular/angular.js"></script>
<script src="node_modules/angular-route/angular-route.js"></script>
<script src="node_modules/angular-messages/angular-messages.js"></script>
<script src="node_modules/angular-animate/angular-animate.js"></script>
<script src="node_modules/angular-aria/angular-aria.js"></script>
<script src="node_modules/angular-material/angular-material.js"></script>
<script src="node_modules/angular-material-data-table/dist/md-data-table.js"></script>
<script src="node_modules/angular-cookies/angular-cookies.js"></script>
<script src="node_modules/firebase/firebase.js"></script>
<script src="node_modules/angularfire/dist/angularfire.min.js"></script>
<script src="node_modules/angular-sanitize/angular-sanitize.js"></script>
<script src="node_modules/ui-select/dist/select.js"></script>

 <script src="bower_components/angular-summernote/dist/angular-summernote.js"></script>

**Comment by aalshaikh** [comment]
Hi @deblis did you fix this issue?

**Comment by deblis** [comment]
I could not fix it because the theme I'm using does not show the
dropdown, but when I try without a theme it works.

2016-12-06 20:24 GMT, alaaalshaikh <notifications@github.com>:

**Comment by aalshaikh** [comment]
I remove duplicated bootstraps JS, and Style dropdown is working now.

**Comment by deblis** [comment]
Exact. Styles that have bootstrap or themes cause the dropdow class to change and then stop working. If you know of a solution for integration can you help me?

**Comment by aalshaikh** [comment]
@deblis i don't how to integrate summernote with your theme exactly because i am using theme already done the integration, maybe if you give more details about what you did maybe i can help you.
